pyvisa/util.py

```diff
@@ -82,6 +83,12 @@ def read_user_library_path():
 
     logger.debug('User defined library files: %s' % files)
     try:
+        try:
+            dependencyPaths = config_parser.get("Paths", "dependency paths")
+            for path in dependencyPaths.split(";"):
+                os.add_dll_directory(path)
```

Comment:
This is available only on windows (os.add_dll_directory docs in https://docs.python.org/3/library/os.html )so I think this whole section should be behind a `sys.platform` check. I can easily see people copy pasting a file from a windows to a linux machine and I would prefer to avoid crahsing because the function is not available.

Reply:
Easy enough, makes sense